libretro-buildbot-recipe.sh

```diff
@@ -238,6 +238,18 @@ buildbot_log() {
 	fi
 }
 
+buildbot_tag() {
+	# If the repository has no tags, it returns an empty string
+	TAG=$(git ls-remote --quiet --tags --sort="v:refname" | tail -n1 | sed 's/.*\///; s/\^{}//')
+
+	if [ "${TAG}" ]; then
```

Comment:
I think the only thing left is to make sure the old tag and new tag are not the same thing. As it stands it will attempt to create a new tag even if the existing tag is exactly the same.

Reply:
Let me fix it :) Thanks for pointing this out

Reply:
Done!